### PR TITLE
rfc27: fix typo

### PR DESCRIPTION
**Pull request by chu11**

### CODE REVIEW

**Comment by chu11** [comment]
noticed 1 other thing I could tweak.  So I removed merge-when-passing until that additional update can be check out!

File: spec_27.rst [hunk 1]
```diff
@@ -343,6 +343,10 @@ The job manager SHALL maintain a dictionary of annotations for each job.
 Each ANNOTATE response and the SUCCESS response (if it contains annotations)
 SHALL update the dictionary according to the following rules:
 
+- If a key exists and is a dictionary, and the new value is a
+  diciontary, the rules below SHALL be applied to the dictionary
```
Comment (garlick) [inline spec_27.rst:347]: spelling

Comment (chu11) [reply spec_27.rst]: doh!  fixed and squashed

**Comment by garlick** [comment]
Feel free to MWP when ready

**Comment by chu11** [comment]
thanks!

**Pull request merged**